webapp/platform/design-system/src/components/primitives/tag/tag.stories.tsx

```diff
@@ -0,0 +1,423 @@
+            <GuestTag size='xs'/>
+            <GuestTag size='sm'/>
+            <GuestTag size='md'/>
+            <GuestTag size='lg'/>
+        </div>
+    ),
+};
+
+// With Icon
+export const WithIcon: Story = {
+    render: () => (
+        <div style={{display: 'flex', gap: '12px', flexDirection: 'column', alignItems: 'flex-start'}}>
+            <Tag text='With Icon' icon={<MockIcon size={12}/>} variant='success' size='xs'/>
+            <Tag text='With Icon' icon={<MockIcon size={16}/>} variant='success' size='sm'/>
+            <Tag text='With Icon' icon={<MockIcon size={18}/>} variant='info' size='md'/>
+            <Tag text='With Icon' icon={<MockIcon size={20}/>} variant='warning' size='lg'/>
+        </div>
+    ),
+};
+
+// Clickable
+export const Clickable: Story = {
+    render: () => (
+        <div style={{padding: '20px', background: '#f0f0f0'}}>
+            <div style={{display: 'flex', gap: '8px', alignItems: 'center', flexWrap: 'wrap'}}>
+                <Tag
+                    text='Click Me'
+                    variant='info'
+                    size='sm'
+                    uppercase={true}
+                    // eslint-disable-next-line no-alert
+                    onClick={() => alert('Tag clicked!')}
+                />
+                <Tag
+                    text='Clickable with Icon'
+                    icon={<MockIcon size={18}/>}
+                    variant='success'
+                    size='md'
+                    uppercase={true}
+                    // eslint-disable-next-line no-alert
+                    onClick={() => alert('Tag with icon clicked!')}
+                />
+            </div>
+        </div>
+    ),
+};
+
+// With Tooltip
+export const WithTooltipStory: Story = {
+    name: 'With Tooltip',
+    render: () => (
+        <div style={{padding: '20px', background: '#f0f0f0'}}>
+            <div style={{display: 'flex', gap: '12px', alignItems: 'center', flexWrap: 'wrap'}}>
+                <WithTooltip title='This is additional information about the tag'>
+                    <Tag
+                        text='Hover Me'
+                        variant='info'
+                        size='md'
+                        uppercase={true}
+                    />
+                </WithTooltip>
+                <WithTooltip title='Additional user context information'>
+                    <Tag
+                        text='User Info'
+                        variant='success'
+                        size='sm'
+                    />
+                </WithTooltip>
+            </div>
+        </div>
+    ),
+};
+
+// Long Text Overflow
+export const LongTextOverflow: Story = {
+    render: () => (
+        <div style={{maxWidth: '200px', padding: '20px', border: '1px dashed #ccc'}}>
+            <Tag
+                text='This is a very long tag text that will overflow'
+                variant='info'
+                size='sm'
+            />
+        </div>
```

Comment:
```suggestion
        <div style={{maxWidth: '200px',}}>
            <Tag
                text='This is a very long tag text that will overflow'
                variant='info'
                size='sm'
            />
        </div>
```
don't think we need the padding/border